ui/app/components/mount-backend-form.js

```diff
@@ -156,11 +157,11 @@ export default class MountBackendForm extends Component {
         'You do not have access to the config endpoint. The secret engine was mounted, but the configuration settings were not saved.'
       );
       // remove the config data from the model otherwise it will save it even if the network request failed.
-      [this.mountModel.maxVersions, this.mountModel.casRequired, this.mountModel.deleteVersionAfter] = [
-        0,
-        false,
-        0,
-      ];
+      [
```

Comment:
The custom component logic will allow us to move this to a KV-mount specific component